flutter/lib/src/file_system_transport.dart

```diff
@@ -11,10 +12,12 @@ class FileSystemTransport implements Transport {
 
   @override
   Future<SentryId?> send(SentryEnvelope envelope) async {
-    final envelopeData = <int>[];
-    await envelope.envelopeStream(_options).forEach(envelopeData.addAll);
-    // https://flutter.dev/docs/development/platform-integration/platform-channels#codec
-    final args = [Uint8List.fromList(envelopeData)];
+    final eventIdLabel = envelope.header.eventId?.toString() ?? '';
+    final args = await compute(
+      _convert,
+      _CaptureEnvelopeData(envelope, _options),
+      debugLabel: 'captureEnvelope $eventIdLabel',
+    );
     try {
       await _channel.invokeMethod<void>('captureEnvelope', args);
```

Comment:
`args` previously was an array 
```suggestion
      await _channel.invokeMethod<void>('captureEnvelope', [args]);
```

Reply:
There should be a comment which tells you which type should get passed. Probably also a test which asserts that 😅

Reply:
that's indeed another problem, but the error is actually with the `compute` flag, See https://github.com/getsentry/sentry-dart/pull/707#issuecomment-1015772386

Reply:
Well, at least the tests are now passing.